sormas-backend/src/main/java/de/symeda/sormas/backend/event/EventParticipantFacadeEjb.java

```diff
@@ -564,8 +564,20 @@ public List<ProcessedEntity> restore(List<String> uuids) {
 		if (eventParticipantsToBeRestored != null) {
 			eventParticipantsToBeRestored.forEach(eventParticipantToBeRestored -> {
 				try {
-					restore(eventParticipantToBeRestored.getUuid());
-					processedEventParticipants.add(new ProcessedEntity(eventParticipantToBeRestored.getUuid(), ProcessedEntityStatus.SUCCESS));
+					final boolean empty = getByEventAndPersons(
```

Comment:
please rename to `isPersonNatParticipant` or similar

Reply:
renamed